ecosystem/typescript/sdk/package.json

```diff
@@ -86,5 +86,5 @@
     "typedoc": "^0.23.20",
     "typescript": "4.8.2"
   },
-  "version": "1.15.0"
+  "version": "1.16.0"
```

Comment:
I been always wondering, is it possible to somehow point this to `version.ts`?

Reply:
haha yes! https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/version.ts#L1
didn't have the time, probably a good ops week item